cli/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ _Released 07/01/2025 (PENDING)_
 - The Cypress configuration wizard for Component Testing supports TypeScript 5.0 or greater. Addresses [#31187](https://github.com/cypress-io/cypress/issues/31187).
 - `@cypress/webpack-dev-server` and `@cypress/webpack-preprocessor` no longer support `webpack` version 4. Addresses [#31344](https://github.com/cypress-io/cypress/issues/31344). If you still need to use `webpack` version 4, please see our [migration guide](https://docs.cypress.io/app/references/migration-guide#Migrating-to-Cypress-150).
 - `@cypress/webpack-dev-server` no longer supports `webpack-dev-server` version 4. Addresses [#31605](https://github.com/cypress-io/cypress/issues/31605). If you still need to use `webpack-dev-server` version 4, please see our [migration guide](https://docs.cypress.io/app/references/migration-guide#Migrating-to-Cypress-150).
+- `@cypress/webpack-batteries-included-preprocessor` no longer ships with browser built-ins that came by default with Webpack 4. Please see [resolve.fallback](https://webpack.js.org/configuration/resolve/#resolvefallback) for a list of available built-ins, as well as the [`@cypress/webpack-batteries-included-preprocessor` README](../npm/webpack-batteries-included-preprocessor/README.md) for how to add built-ins to the preprocessor. Addresses [#31039](https://github.com/cypress-io/cypress/issues/31039).
```

Comment:
This original description seemed a bit confusing referring to the list of resolve.fallback upfront - like it was a reference for what? Making the changelog more explicit of the action in the front of the sentence.

```suggestion
- The `@cypress/webpack-batteries-included-preprocessor` no longer includes browser built-ins that were automatically provided by Webpack 4. To add built-ins manually, refer to the Webpack [resolve.fallback](https://webpack.js.org/configuration/resolve/#resolvefallback) documentation and the [`@cypress/webpack-batteries-included-preprocessor` README](../npm/webpack-batteries-included-preprocessor/README.md). Addresses [#31039](https://github.com/cypress-io/cypress/issues/31039).
```